pkg/cmd/roachprod/vm/gce/gcloud.go

```diff
@@ -289,6 +291,7 @@ func (o *providerOpts) ConfigureCreateFlags(flags *pflag.FlagSet) {
 			"will be repeated N times. If > 1 zone specified, nodes will be geo-distributed\n"+
 			"regardless of geo (default [%s])",
 			strings.Join(defaultZones, ",")))
+	flags.BoolVar(&o.preemptible, ProviderName+"-preemptible", false, "use preemptible instances")
```

Comment:
Can we make it clear this is just for GCP clusters?

Reply:
In the help message? The actual flag printed by `--help` is `--gce-preemptible`.

Reply:
Added.